deploy/helm/flightctl/templates/flightctl-cli-artifacts-deployment.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if .Values.cliArtifacts.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    flightctl.service: flightctl-cli-artifacts
+  name: flightctl-cli-artifacts
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      flightctl.service: flightctl-cli-artifacts
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        flightctl.service: flightctl-cli-artifacts
+    spec:
+      containers:
+        - name: flightctl-cli-artifacts
+          image: {{ .Values.cliArtifacts.image.image }}:{{ default .Chart.AppVersion .Values.cliArtifacts.image.tag }}
+          imagePullPolicy: {{ .Values.cliArtifacts.image.pullPolicy }}
+          ports:
+            - containerPort: {{ int .Values.global.nodePorts.cliArtifacts }}
```

Comment:
Actually I think we dont want to make the `containerPort` configurable. We don't to it for other fctl deplyments neither.
The same applies for the `flightctl-cli-artifacts-service`. The only configurable port should be `nodePort`.
All other ports should be static as it makes it much easier to reference the well-known service name + port.

Reply:
done

Reply:
done